rules/falco_rules.yaml

```diff
@@ -3010,6 +3010,12 @@
   output: Drift detected (open+create), new executable created in a container (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline filename=%evt.arg.filename name=%evt.arg.name mode=%evt.arg.mode event=%evt.type)
   priority: ERROR
 
+# find when a new kernel module is injected
+- rule: Linux Kernel Module injection detected
+  desc: It is very uncommon for kernel modules to be injected in running production instances, used rookits to obfuscate their behavior via kernel hooking 
+  condition: evt.type=execve and proc.name=insmod
```

Comment:
I would suggest to limit to activities happen from inside a container. So the condition would be something like:
`spawned_process and container and proc.name=insmod`

Reply:
Agreed with @leogr , need to have a whitelisted images/containers that perform kernel module injection like falco.

Reply:
I'm @leodido anyway 😄

Reply:
> I'm @leodido anyway

I believe @Kaizhe referred to [my comment](https://github.com/falcosecurity/falco/pull/1401#pullrequestreview-504589976) where I was suggesting to introduce a `list` :smile_cat: 

Anyway, to summarize, I believe we all agree on two things:
- a list of known images/containers that perform kernel module injection is needed
- the above list must include Falco's images